Week3/More JavaScript.js

```diff
@@ -132,11 +160,16 @@ console.log("Amazing Joe's Garage, we service " + listOfVehicle[0] + "s" + nameO
 // 13. Create an object that contains the teachers that you have had so far for the different modules.
 // 14. Add a property to the object you just created that contains the languages that they have taught you.
 
-    class23.teachers = 'Philipp, Rob, Unmesh & Bonan, Yash';
+class23.teachers = ['Philipp', 'Rob', 'Unmesh & Bonan', 'Yash'];
 
-    class23.languages = 'Html, CSS, Git, JavaScript1'
- 
-    console.log(class23); // {teachers: "Philipp, Rob, Unmesh & Bonan, Yash", languages: "Html, CSS, Git, JavaScript1"}
+class23.languages = ['Html', 'CSS', 'Git', 'JavaScript1'];
```

Comment:
Ah, it is better now! But I actually made a mistake on my first review of the code. 
It is better that `teachers` and `languages` are an array, but now it is difficult to know which teacher taught which language?

So for example, if `Philipp` taught `HTML` and `CSS`, now there is no way to know that. Hint: You could use an array with objects in it, with arrays as properties, like this:
```
const objects = [
{
 name: 'someone',
 hobbies: ['first', 'second']
},
{
 name: 'someone2',
 hobbies: ['first', 'second']
},
]
```

Do you think you could change your code to look more like that?

Reply:
I redesigned the code using the nested objects.